.github/workflows/test-templates.yaml

```diff
@@ -1,10 +1,10 @@
-name: Func Unit Test
```

Comment:
I do not get this rename. This is foremost "unit test". It also runs "template test" but that is IMO secondary.